### ISSUE TITLE
docs: Documentation does not link to possible values for configuration keys

### ISSUE THREAD

**Issue by rptb1** [canonical/lxd#12528]
*Please describe the question or issue you're facing with "lxc storage volume set - Canonical LXD documentation".*

In common with most LXD documentation, the page describing a command does not lead to documentation of the possible arguments for that command.

In this case, we see that `lxc storage volume set` can take `key=value` pairs.  But there is no description of what these values could be.  Nor is there any link or reference to that information, or description of where to find it.

The same problem applies to all commands.  Another example is https://documentation.ubuntu.com/lxd/to/latest/reference/manpages/lxc/config/set/#lxc-config-set-md although there *is* a link in the "howto" section at https://documentation.ubuntu.com/lxd/to/latest/howto/instances_configure/

I think a cross-reference to all possible values for parameters needs to be added to every command reference page.

I also think that the output of help commands (e.g. `lxc help storage volume create`) should have a sentence indicating how to get this information.

By the way, in the case of this one page, there is a sentence at https://documentation.ubuntu.com/lxd/to/latest/howto/storage_volumes/#configure-storage-volume-settings that could be copied: "See the [Storage drivers](https://documentation.ubuntu.com/lxd/to/latest/reference/storage_drivers/#storage-drivers) documentation for the available configuration options for each storage driver."  But as I say, this is a problem across many pages.

Thanks!
---
*Reported from: https://documentation.ubuntu.com/lxd/to/latest/reference/manpages/lxc/storage/volume/set/

**Comment by ru-fu** [comment]
Yes, I agree that adding these references would be very helpful!

It's a little tricky though, since we automatically generate the documentation for the man pages from the `lxc` command's help texts - so we'll need to update the help texts in such a way that the links work both from the command line (probably pointing to the online version of the docs) and within the documentation.

**Comment by tomponline** [comment]
The auto completion in the `lxc` CLI tool now uses the LXD metadata API to suggest config keys that the remote server supports.
